src/modules/store.js

```diff
@@ -48,5 +48,21 @@ class Store {
     });
     localStorage.setItem('tasks', JSON.stringify(tasks));
   }
+
+  static updateCheckbox(index, completed) {
+    const tasks = Store.getTasks();
+    tasks.forEach((task) => {
+      if (task.index == index) {
+        task.completed = completed;
+      }
+    });
+    localStorage.setItem('tasks', JSON.stringify(tasks));
+  }
+
+  static clearCompletedTasks() {
+    const tasks = Store.getTasks();
+    const filteredTasks = tasks.filter((ele) => ele.completed != true);
+    localStorage.setItem('tasks', JSON.stringify(filteredTasks));
+  }
```

Comment:
- There are three items in the todo list such as: `one` which id is `1`, `two` which id is `2` and `three` which id is `3`.
![image](https://user-images.githubusercontent.com/60083437/236589858-500ae645-9856-436a-90cc-12b6885e7298.png)
- I have removed the item `two` which id was `2`. So as per the project requirement id of the next item `three` should be `2` now, cause there are only 2 items now and after filter the `id` of every item should be updated.
![image](https://user-images.githubusercontent.com/60083437/236589886-d723e166-b6bf-4596-ab68-a57e8004338a.png)

- [ ] Kindly fix that issue.